docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,382 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+
+```
+kernel.core_pattern=
+kernel.core_uses_pid=0
+```
+
+### D-Bus notification
+
+After the report is generated, crash-manager sends the D-Bus signal with information about the crashed process.
+
+Signature: `sssssiia{sv}`  
+Member: `ProcessCrashed`  
+Interface: `org.tizen.system.crash.Crash`  
+Path: `/Org/Tizen/System/Crash/Crash`  
+Data structure:  
+
+Data         | Type
+-------------|-------
+command name | STRING
+command path | STRING
+appid        | STRING
+pkgid        | STRING
+report path  | STRING
+PID          | INT32
+TID          | INT32
+array        | ARRAY
+
+Array containing pairs of keys (STRING) and values (VARIANT). Possible keys:
+
+Key          | Type   | Description
+-------------|--------|---------------
+x86.eip      | UINT32 | instruction pointer
+x86_64.rip   | UINT64 | instruction pointer
+arm.pc       | UINT32 | instruction pointer
+arm.lr       | UINT32 | instruction pointer
+aarch64.pc   | UINT64 | instruction pointer
+aarch64.lr   | UINT64 | instruction pointer
+sys.signal   | INT32  | signal that caused crash
+sys.tid.comm | STRING | thread name
+
+Depending on the architecture, array will contain:
+
+Arch    | PC (instruction pointer) | Link Register
+--------|--------------------------|------------------
+x86     | x86.eip                  | (not available)
+x86_64  | x86_64.eip               | (not available)
+arm     | arm.pc                   | arm.lr
+aarch64 | aarch64.pc               | aarch64.lr
+
+### minicoredumper
+
+minicoredumper is a tool that accepts a core dump on a standard input and saves the modified (size-reduced) core dump in a file. This tool is internally used by crash-manager to generate minicoredump. Modifications depends on the configuration, and they mainly rely on minimization of the core dump file.
+
+Usually, integrator does not need to modify this configuration unless integrator needs to change:
+
+-   parameters of minicoredump files
+-   for which applications minicoredump is created and for which regular core dump is created
+
+minicoredumper can be used alone by setting /proc/sys/kernel/core_pattern to `|/usr/bin/minicoredumper %P %u %g %s %t %h %e`. All possible % specifiers are described in the [core(5)](http://man7.org/linux/man-pages/man5/core.5.html)).
+
+The default path of the configuration file is `/etc/minicoredumper/minicoredumper.cfg.json` and the file format is JSON. Possible options are:
+
+-   **base_dir** (string): Defines the root directory where the dumped data will be stored. This options is overwritten by command line option which is set by the **crash-manager**.
+-   **watch** (array): A set of recepts, which will be used for applications that meets the defined criteria:
```

Comment:
Please validate is it recepts?

Reply:
I am convinced that yes: https://linutronix.de/minicoredumper/man/man5/minicoredumper.recept.json.5.html